Behat/Context/DrupalExtendedContext.php

```diff
@@ -334,4 +334,23 @@ public function createNodeAuthoredCurrentUser($type, TableNode $nodesTable) {
       $this->nodeCreate($node);
     }
   }
+
+  /**
+   * Wait for AJAX to finish.
+   *
+   * @param int $seconds
+   *   Max time to wait for AJAX.
+   *
+   * @Given I wait for AJAX to finish almost :seconds seconds
```

Comment:
Please change the step from

> @Given I wait for AJAX to finish almost :seconds seconds

To 

> @Given I wait for AJAX to finish at least :seconds seconds

Almost means 'very nearly', I think  'at least' is better.